research/picai/nnunet_scripts/eval.py

```diff
@@ -1,26 +1,188 @@
 import argparse
+import contextlib
+import os
+import warnings
+from collections import defaultdict
+from os.path import join
+from typing import Any, Dict, Iterable, List, Optional
 
-from picai_eval import evaluate_folder
+import numpy as np
+from numpy.typing import NDArray
+
+with contextlib.redirect_stdout(open(os.devnull, "w")):
```

Comment:
If we don't log this to the console, should we maybe redirect to a file or something? Suppressing it entirely may make it difficult to catch things if they pop up in picai_eval

Reply:
I think this is a great suggestiong. Especially since there's quite a few things I'm silencing under the nnunet umbrella (at the moment as a lot of these packages have reminders to cite them printed to console). Id say we leave it as is for now and make log file generation a seperate PR. There is some complexity in terms of figuring out where to place the log file. Placing it in the working directory feels a bit messy. Ideally we could place it in the output directory, but we receive that output directory as an argument. So we will run into not only mypy errors but potentially some import errors when using multiprocessing if we import picai_eval within a method after we know the output folder.